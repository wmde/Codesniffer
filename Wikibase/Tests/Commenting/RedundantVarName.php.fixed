<?php

class Example {

	/**
	 * @var bool
	 */
	private $uncommented;

	/**
	 * @var bool This is just text and should not be reported as an error.
	 */
	private $validComment;

	/**
	 * @var bool $mentioningOtherVariables should be allowed in such a comment.
	 */
	private $someOtherVariableName;

	/**
	 * @var bool Also, $mentioning the same variable name later in the sentence should be allowed.
	 */
	private $mentioning;

	/**
	 * @var bool $notTheSameVariableName should be allowed.
	 */
	private $not;

	/**
	 * @Var bool
	 */
	private $redundantVariableName;

	/**
	 * @var bool after the variable name.
	 */
	private $redundantWithComment;

	/**
	 * @var bool
	 */
	private $redundantBeforeType;

	private function __construct() {
		/** @var bool $local */
		$local = true;
	}

}
