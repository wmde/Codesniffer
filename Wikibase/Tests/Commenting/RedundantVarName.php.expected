 31 | ERROR | [x] Found redundant variable name "$redundantVariableName" in @var
 36 | ERROR | [x] Found redundant variable name "$redundantWithComment" in @var
 41 | ERROR | [x] Found redundant variable name "$redundantBeforeType" in @var
